docs/docs/rules.mdx

```diff
@@ -54,6 +54,28 @@ rules:
 This example rule applies at the start of conversation as well as when the user decides
 to a send a message with an intent `greet` in the middle of an ongoing conversation.
 
+After training, `RulePolicy` will perform a check to make sure that the rules don't
+contradict the stories and each other.
+For example the following two rules are contradicting:
```

Comment:
should this be documented here and not under rule policy?

Reply:
what do you mean by `here`?

Reply:
where it's documented currently - i don't understand why it's here and not under the RulePolicy page

Reply:
From the look of it. This whole file describes the RulePolicy, so I added it here, I can move it to policies.mdx

Reply:
Relevant to the comment I left in policies.mdx, this should be moved to a subsection under `RulePolicy` that includes a definition + example of contradicting stories.